### ISSUE TITLE
Migrate and upstream LowLatencyAudio plugin

### ISSUE THREAD

**Issue by timwindsor** [blackberry/WebWorks-Community-APIs#378]
Our LowLatencyAudio plugin is based on a similar API as this one, which I've forked into the BlackBerry account: https://github.com/blackberry/cordova-plugin-lowlatencyaudio

We should be able to migrate our code into it and submit it upstream so that the main plugin can support BlackBerry 10 in addition to the other platforms.

**Comment by umcsdon** [comment]
I can work on this one.

Note:
#341 was already merged into the upstream, so we would able to pull it.
For #336 , I'm waiting for my micro-hdmi cable to arrive (will work on it shortly).

**Comment by umcsdon** [comment]
Well...The upstream uses audio file with mp3 extension. OpenAL can be used for audio file with wav extension (decode required in order to play mp3).

I have to take a look at the native mp3 example first.

**Comment by timwindsor** [comment]
Yeah, that's quite a bit of modification to do. Hopefully you can find some decent libraries on device or in open source to make it manageable.
